kolibri/plugins/learn/assets/src/views/LibraryPage/index.vue

```diff
@@ -49,16 +49,20 @@
           v-else-if="!displayingSearchResults && !rootNodesLoading"
           data-test="channels"
         >
-          <h1 class="channels-label">
-            {{ channelsLabel }}
-          </h1>
-          <p
-            v-if="isLocalLibraryEmpty"
-            data-test="nothing-in-lib-label"
-            class="nothing-in-lib-label"
-          >
-            {{ coreString('nothingInLibraryLearner') }}
-          </p>
+          <NoResourcePage v-if="isChannelEmpty && isLearner" />
```

Comment:
I think we probably want to conditionally show this based on whether there are any libraries around you or not. If there are, we should show the existing message below, if not, then we should show this component.

This will require us to communicate something out of the OtherLibraries component to dynamically update this based on whether any libraries are being shown.

Reply:
Conditionalizing this on being a learner means that in the completely empty state, we're not showing proper messaging to an admin to prompt them to add resources to the device:

![image](https://github.com/user-attachments/assets/b4a617e0-cfb4-4221-93c8-e1a764d28332)